CRM/Utils/System/Drupal.php

```diff
@@ -852,4 +852,51 @@ public function onCiviExit() {
     }
   }
 
+  /**
+   * @inheritdoc
+   */
+  public function hasUsersTable():bool {
```

Comment:
This one should be in DrupalBase.php. (Sidenote: I agree the whole user tablename thing is weird.)

Reply:
Done.